api/tests/mocha/controllers/record-utils-parser.spec.js

```diff
@@ -10,14 +10,14 @@ describe('record-utils-parser', () => {
     sinon.restore();
   });
 
-  it('assert month is parsed as integer', () => {
+  it('assert month is parsed into a string using a list', () => {
     sinon.stub(config, 'get').returns(definitions.forms);
     const body = {
       from: '+888',
       message: '1!YYYY!facility#2011#11'
     };
     const doc = recordUtils.createByForm(body);
-    chai.expect(11).to.equal(doc.fields.month);
+    chai.expect('November').to.equal(doc.fields.month);
```

Comment:
this is what, according to the test code, should *actually* happen, because we are declaring a list mapping against `YYYY` (see the `form-definitions.js` file). It wasn't happening before because another feature, which is not used, was masking it.

FWIW the `list` feature isn't documented but it looks like some projects are using it, so we can't delete it yet